### ISSUE TITLE
Cannot mount Appimage, please check your FUSE setup

### ISSUE THREAD

**Issue by xXWhacker** [AppImage/AppImageKit#1143]
Every time I launch Zoho mail Appimage Iget the below. Normally when Zoho is installed I'll get the message 2 or 3 times and then never again, but this tiome the message just isn't going away even though Zoho is launching with what seems no issues. Running AcroLinuxD Plasma desktop using the 1.20 version of the Zoho mail Appimage. Any help would be appreciated, Thanks.

Nothing here seems to resolve the issue:

https://github.com/AppImage/AppImageKit/wiki/FUSE

```
 You might still be able to extract the contents of this AppImage 
if you run it with the --appimage-extract option. 
See https://github.com/AppImage/AppImageKit/wiki/FUSE 
for more informatio
```

**Comment by probonopd** [comment]
Can you run other AppImages?

**Comment by xXWhacker** [comment]
Acro is a Arch based OS. I can't speak to Ubuntu, Debian, or other OS's but with every Arch based OS with Zoho you click to add it to to the system and the first 3 or 4 attempts to launch it you get the fuse message, then after that you never see the message again and Zoho runs fine. Zoho is working as it should but this time every time it's run the fuse message comes up. Now I just CPU-X and ran it twice and no issue / message. Looks like it's a Zoho specific issue.

**Comment by probonopd** [comment]
Looks like it. Please report it to the Zoho developers then, thank you.

**Comment by drbonesaw** [comment]
Sorry for the late reply but I'm not running Ubuntu just to test this for Whacker. I am running Garuda which is also Arch based and at about 3:15 AM PST I just got this. From what I remember of the updates FUSE was updated 3 or 4 days ago, so why this would be popping up now I have no idea.  Now back to Ubuntu can't use it to  narrow this down to a Arch issue, cause the Zoho Appimage once installed on the first few attempts top run has always given this message then just goes away. I  am going to point the Zoho devs at this issue ticket since I don't see any other Appimage doing this.
